src/main/java/net/mcreator/ui/workspace/selector/WorkspaceSelector.java

```diff
@@ -406,20 +415,43 @@ private void reloadRecents() {
 			reloadRecents();
 	}
 
-	private void addWorkspaceButton(String text, ImageIcon icon, ActionListener event, JPanel container) {
+	private JButton mainWorkspaceButton(String text, ImageIcon icon, ActionListener event) {

```

Comment:
Is there a need to keep this as a separate method for only 1 usage? Just to make the code cleaner?

Reply:
Mostly to isolate creation of the button from other code